userspace/libsinsp/parsers.cpp

```diff
@@ -159,7 +159,7 @@ void sinsp_parser::process_event(sinsp_evt *evt)
 		{
 			sinsp_evt_param *parinfo = evt->get_param(0);
 			uint16_t evid = *(uint16_t *)parinfo->m_val;
-			flags = g_infotables.m_syscall_info_table[evid].flags;
+			flags = EF_NONE;
```

Comment:
All flags were already EF_NONE in any case.

Reply:
This looks like the kind of change that is small enough to be ignored, yet can bring headaches to other people in the future. Could you elaborate on why do we need this change? If the only reason is the one you already provided, I would roll this back.

Reply:
Well, we do not need flags in syscall info table; when they were present (and they are not present since some months now) they were just using the `EF_DROP_SIMPLE_CONS` one, that is now useless.
I think it creates more tech debt than its provided value (that is near 0 btw :D )

Reply:
Then IMO the flags should be removed entirely and not silently escaped in a single place, this way we still have the flags field floating around, but it is not doing anything and can lead people to believing it is still being used. I would roll this back and open a PR properly removing flags entirely, would that make sense?

Reply:
Mmh there is no `flags` field anymore:
```
struct ppm_syscall_desc {
	enum ppm_event_category category; /**< System call category. */
	char name[PPM_MAX_NAME_LEN]; /**< System call name, e.g. 'open'. */
};
```
I think i am misunderstanding something here :D

Reply:
I missed that, it's still early for me, guess I need more coffee.

Reply:
ahahahah no problem my man! :coffee:

Reply:
Then how about initializing the variable when declared in line 156 and removing this line entirely? That would make it really explicit that flags are no longer used in the case of generic events.

Reply:
Yeah that's true, but at the same time it would be less explicit about handling the special `generic` case. I mean, if 2 months from now one decides to remove the initialization, we would end up reading garbage in case of generic events.
I don't know, it's a tradeoff :)